packages/central-server/src/tests/apiV2/changes/getChangesMetadata.test.js

```diff
@@ -140,7 +140,7 @@ describe('GET /changes/metadata', async () => {
         });
 
         const { changeCount, countries } = response.body;
-        expect(changeCount).to.equal(8);
+        expect(changeCount).to.equal(10);
```

Comment:
Strictly speaking we don't need to sync down the 2 new records here. Since we've previously synced against any country, we should know we've synced them. But I didn't really wanna go into the depths of that logic as it is kinda complicated, and there's no real harm in a little 'over syncing'.

Ideally we switch from this 'NULL countryIds means all countries' logic to a proper wild card and then it should be safer to clean such things up 👍